pkg/snowflake/parameters.go

```diff
@@ -779,16 +864,35 @@ func ShowObjectParameter(db *sql.DB, key string, objectType ObjectType, objectId
 	return &value, nil
 }
 
-func ListParameters(db *sql.DB, parameterType ParameterType, pattern string) ([]Parameter, error) {
+func ListAccountParameters(db *sql.DB, pattern string) ([]Parameter, error) {
 	var stmt string
-	if parameterType == ParameterTypeAccount || parameterType == ParameterTypeSession {
-		if pattern != "" {
-			stmt = fmt.Sprintf("SHOW PARAMETERS LIKE '%s' IN %v", pattern, parameterType)
-		} else {
-			stmt = fmt.Sprintf("SHOW PARAMETERS IN %v", parameterType)
+	if pattern != "" {
+		stmt = fmt.Sprintf("SHOW PARAMETERS LIKE '%s' IN ACCOUNT", pattern)
+	} else {
+		stmt = "SHOW PARAMETERS IN ACCOUNT"
+	}
+	log.Printf("[DEBUG] query = %s", stmt)
```

Comment:
Does that show up in the terraform output?

Reply:
only if debug logs are enabled